packages-native/web-view/src/WebView.tsx

```diff
@@ -38,6 +38,16 @@ export class WebView extends Component<Props> {
                     onLoad={this.onLoadHandler}
                     onError={this.onErrorHandler}
                     userAgent={this.props.userAgent}
+                    onShouldStartLoadWithRequest={event => {
```

Comment:
event is just being used to check url, can you instead of use event, use `({url}) => ` ?

Reply:
Good point, cleaner code, done.